### ISSUE TITLE
Scrapper should suck-in PDF text as well as background images

### ISSUE THREAD

**Issue by powerbroker** [AriZoneVibes/ServLibScrapper#1]
Feed scrapper with https://servlib.com/panasonic/telephone/kx-ts2351rub-kx-ts2351ruw.html and compare with the source preview.

There are tons of text labels missing in the resulting PDF.

The ServLibFags render PDF text in `<div class="pagedoc"/>` tag of their pages. There are ~3 PDF pages rendered when URL has no "?start=${page_number}" parameter, and single PDF page otherwise.

Would be great if Scrapper start with manual web page and suck-in all the required HTML pages(the fags render total amount of PDF pages on their first page in the `<dt>Pages</dt><dd>51</dd>` construct inside `<div class="file_desc"\>`), extract `<div class="pagedoc"/>` from them and join extracted HTML with the backgrounds.

The resulting document may be (zipped) HTML instead of PDF

**Comment by AriZoneVibes** [comment]
Hi ^^
Glad you are finding use to the script. I got to understand what the issue is. At the time, I didn't realize ServLib stores them in different ways, in this case as PDF instead of PNG files. 
I'll work on finding how to also grab that information from the website. Good idea with exporting it as HTML

**Comment by robertschulze** [comment]
Maybe easier to automate the retrieval in ChromeDriver using the integrated screenshot functionality on the <div> tag(?)

**Comment by robertschulze** [comment]
Yes, that's true but it can be (re)detected using a free online PDF OCR service or, if it has to be in Python, using tesseract.
I fear that trying to reconstruct from the HTML and background will be a lot of effort (getting the size, positioning, etc. correct). What could be an alternative is printing the website to PDF, which should maintain the text, and then cropping to the actual manual page.
